module/zfs/spa.c

```diff
@@ -11074,6 +11194,12 @@ ZFS_MODULE_PARAM(zfs_livelist_condense, zfs_livelist_condense_, new_alloc, INT,
 	"Whether extra ALLOC blkptrs were added to a livelist entry while it "
 	"was being condensed");
 
+ZFS_MODULE_PARAM(zfs_spa, spa_, note_txg_time, UINT, ZMOD_RW,
+	"How often txg should be registred in database");
```

Comment:
"registered"

Or maybe: "How frequently TXG timestamps are stored internally for scrubs (in seconds)"

Reply:
Done.